### PR TITLE
added-notes-projects-files

### PR DESCRIPTION
**Pull request by OSAMA478**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+	height: 480px;
+	margin: 0 auto;
+	background-color: #fafafa;
+	box-shadow: 5px 10px 20px rgb(0, 0, 0, 0.4);
+	border-radius: 20px;
+	overflow: hidden;
+}
+
+.paper::before {
+	content: "";
```
Comment (Rehan-Khan-Engineer) [inline style.css:41]: Group Similar properties @OSAMA478

**Review by Rehan-Khan-Engineer — CHANGES_REQUESTED** [review]
Group similar properties @OSAMA478

File: style.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height: 100vh;
+}
+
+.paper {
+	background-color: aqua;
+	position: relative;
+	/* width: 90%; */
```
Comment (Rehan-Khan-Engineer) [inline style.css]: You can remove commented out code.

Comment (OSAMA478) [reply style.css]: line removed @Rehan-Khan-Engineer

File: style.css (hunk 1 above)
Comment (OSAMA478) [reply style.css:41]: I have grouped similar properties.

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]
Excellent @OSAMA478

File: style.css (hunk 1 above)
Comment (Rehan-Khan-Engineer) [reply style.css:41]: Great @OSAMA478

**Review by Rehan-Khan-Engineer — APPROVED** [review]

**Review by faran4hatch — CHANGES_REQUESTED** [review]
It looks a little off center. Can you keep it in the center !

**Review by anask601 — APPROVED** [review]
Great work 💯 @OSAMA478

**Pull request merged**